Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -1334,7 +1337,7 @@ UpdateSmmInfo (
   SmmInfoHob->SmmBase = MmioRead32 (TO_MM_PCI_ADDRESS (0x00000000) + TSEG) & ~0xF;
   SmmInfoHob->SmmSize = MmioRead32 (TO_MM_PCI_ADDRESS (0x00000000) + BGSM) & ~0xF;
   SmmInfoHob->SmmSize -= SmmInfoHob->SmmBase;
-  SmmInfoHob->Flags = 0;
+  SmmInfoHob->Flags   = SMM_FLAGS_4KB_COMMUNICATION;
```

Comment:
Why is the flag set?

Reply:
if this flag is set, UEFI payload will put SMM rebase information to a 4KB communication region, so in S3 path, SBL will restore SMM setting by these information. So it is required by UEFI SMM feature to support S3.